auth/auth.go

```diff
@@ -171,6 +171,13 @@ func (a Authenticator) Token(ctx context.Context, state string, r *http.Request,
 	return a.config.Exchange(ctx, code, opts...)
 }
 
+// Return a new token if an access token has expired.
+// If it has not expired, return the existing token.
+func (a Authenticator) RefreshToken(ctx context.Context, token *oauth2.Token) (*oauth2.Token, error) {
```

Comment:
I'm not sure calling this `RefreshToken` aligns with the GoDoc comment. Perhaps `GetToken` would be a clearer name - since the refreshing only occurs in certain circumstances.

Reply:
@strideynet 
Thank you for you opinion, and I understand what you are saying. 

One concern I have with naming it `GetToken` is that new users of this library may mistakenly think that they can get *oauth2.Token just by calling the `GetToken` function. (In fact, users need to call [Authenticator `Token` method](https://github.com/zmb3/spotify/blob/dc95f4def888572a00e6a3ae446d09b45cc914aa/auth/auth.go#L158-L172) to get the token for the first time.)

What do you think about this?

Reply:
I see what you are saying - and agree. Let's move forwards with this.